symengine/basic.h

```diff
@@ -8,17 +8,13 @@
 #define SYMENGINE_BASIC_H
 
 // Include all C++ headers here:
-#include <cstddef>
-#include <stdexcept>
-#include <iostream>
```

Comment:
We should add these as the presence of these headers in others are not guaranteed